newrelic/hooks/external_botocore.py

```diff
@@ -224,6 +225,8 @@ def create_chat_completion_message_event(
             "vendor": "bedrock",
             "ingest_source": "Python",
         }
+        if all_token_counts:
+            chat_completion_message_dict["token_count"] = 0
```

Comment:
Is this the value that was previously added in the pipeline? Are we just setting it to 0 as a transition?